specification/postgresql/resource-manager/readme.md

```diff
@@ -90,6 +90,16 @@ input-file:
 - Microsoft.DBforPostgreSQL/preview/2017-12-01-preview/postgresql.json
 ```
 
+### Tag: package-2020-02-14-privatepreview
```

Comment:
As convention in this file, you might want to put this part to the top -- latest version on the top.

Reply:
move this to top.